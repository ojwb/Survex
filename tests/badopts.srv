; pos=fail err=3 warn=0 dummycrprotection=
#units save restore restore restore ; `restore` without corresponding `save`
#units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
#units order=bad typeab=bad typevb=bad
