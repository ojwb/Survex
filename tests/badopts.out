./badopts.srv:2:21: error: No matching SAVE
 #units save restore restore restore ; `restore` without corresponding `save`
                     ^~~~~~~
./badopts.srv:2:29: error: No matching SAVE
 #units save restore restore restore ; `restore` without corresponding `save`
                             ^~~~~~~
./badopts.srv:3:10: error: Expecting "F" or "M"
 #units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
          ^
./badopts.srv:3:16: error: Expecting "D", "G", or "M"
 #units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
                ^
./badopts.srv:3:22: error: Expecting "D", "G", or "M"
 #units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
                      ^
./badopts.srv:3:29: error: Expecting "D", "G", or "M"
 #units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
                             ^
./badopts.srv:3:35: error: Expecting "D", "G", "M", or "P"
 #units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
                                   ^
./badopts.srv:3:42: error: Expecting "D", "G", "M", or "P"
 #units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
                                          ^
./badopts.srv:3:48: error: Expecting "F" or "M"
 #units D=bad a=bad a=bad ab=bad v=bad vb=bad s=bad
                                                ^
./badopts.srv:4:14: error: Data style "bad" unknown
 #units order=bad typeab=bad typevb=bad
              ^~~
./badopts.srv:4:25: error: Expecting "C" or "N"
 #units order=bad typeab=bad typevb=bad
                         ^
./badopts.srv:4:36: error: Expecting "C" or "N"
 #units order=bad typeab=bad typevb=bad
                                    ^
cavern: error: No survey data
